.github/workflows/build-and-test.yml

```diff
@@ -39,6 +39,6 @@ jobs:
       # multiple arguments to it through MSBuild.
       run: |
         for seed in 1 2 3 4 5; do
-          (cd src && dafny /compile:0 /timeLimit:30 /verificationLogger:csv /randomSeed:$seed Main.dfy)
+          (cd src && dafny /compile:0 /timeLimit:30 /verificationLogger:csv /randomSeed:$seed *.dfy)
```

Comment:
Better, but this could potentially break if we ever add any subdirectories of `src` in the future.

The common trick I've seen in Makefiles is `find $(SOURCEDIR) -name '*.dfy'`, could we use that here instead?

Reply:
We could also do `/verifyAllModules Main.dfy` if we expect everything would be imported by it.